web/gui/dashboard_info.js

```diff
@@ -4201,7 +4201,35 @@ netdataDashboard.context = {
     },
 
     // ------------------------------------------------------------------------
-    // WMI
+    // WMI (Process)
+
+    'wmi.processes_cpu_time': {
+        info: 'Elapsed time that all of the threads of this process used the processor to execute instructions by mode (privileged, user). An instruction is the basic unit of execution in a computer, a thread is the object that executes instructions, and a process is the object created when a program is run. Code executed to handle some hardware interrupts and trap conditions is included in this count.'
```

Comment:
I doubt we need to add details about instructions and objects.

> Total CPU utilization. The amount of time spent by the process in [user and kernel](https://en.wikipedia.org/wiki/CPU_modes#Mode_types) modes.

Reply:
Done.